modules/configuring-huge-pages.adoc

```diff
@@ -10,43 +14,66 @@ Node Tuning Operator to allocate huge pages on a specific node.
 tuned profile, which describes how many huge pages should be allocated:
 +
 ----
-$ oc label node <node-using-hugepages> hugepages=true
+$ oc label node <node_using_hugepages> hugepages=true
 ----
 
-. Create a tuned profile for this specific node and allocate some huge pages. To
-associate the profile to the correct node, edit the tuned resource and add a
-`profile` and `match` section:
+. Copy the default tuned profile and customize it to create a tuned profile
+allocating huge pages for your labeled node.
+
+.. Export the default tuned profile and save it to a file (`tuned.yaml`):
 +
 ----
-$ oc edit tuned
+$ oc get tuned --export -o yaml > tuned.yaml
```

Comment:
Missing projects to get tuned.
Moreover I thing is good to change name of exported file to something like hugepages_tuning.yaml
`oc get tuned --export -o yaml -n openshift-cluster-node-tuning-operator > tuned.yaml`